### PR TITLE
Utopia 259: Create Header and Footer common components

### PR DESCRIPTION
**Pull request by Bottle7**
# Description

This PR includes the following proposed change(s):

- Creates and styles Header and Footer components
- Imports fontawesome icon packages for react
- Adds horizontal bc gov logo
- Tweaks sass files to more closely resemble figma wireframes

81A074D1-32B3-4421-8119-974A60C1EEED

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Spun up the frontend container with the new changes.

## Development Dependency Working Agreement
- [ ] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [x] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [x] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

### CODE REVIEW

**Review by binggaofw — APPROVED** [review]

**Comment by kushal-arora-fw** [comment]
Hi @Bottle7 - Can you please add a screenshot of this changes as well? will help visualize better.

**Comment by Bottle7** [comment]
> Hi @Bottle7 - Can you please add a screenshot of this changes as well? will help visualize better.

I have now added a screenshot to the PR description. The additions are the top header bar and footer.

File: src/frontend/src/components/common/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            <li><a>Disclaimer</a></li>
+            <li><a>Accessiblility</a></li>
+            <li><a>Copyright</a></li>
+            <li><a>Contact us</a></li>
+        </ul>
+    </footer>
+  )
+}
+
+export default Footer;
```
Comment (kushal-arora-fw) [inline src/frontend/src/components/common/Footer.tsx:18]: Missing empty line at the end.

**Review by kushal-arora-fw — APPROVED** [review]
Looks good! Thanks for sharing the screenshot. Added a minor comment regarding linting.

File: src/frontend/src/components/common/Footer.tsx (hunk 1 above)
Comment (Bottle7) [reply src/frontend/src/components/common/Footer.tsx]: Thanks, I also caught one in the Header component.

**Pull request merged**